nservicebus/databus.md

```diff
@@ -0,0 +1,69 @@
+---
+title: DataBus feature
+summary: DataBus feature
+tags:
+- DataBus
+- Attachments
+---
+
+## What is DataBus feature for?
+
+Messages are intended to be small. Some scenarios require to send large binary data along with a message. For this purpose, NServiceBus has a DataBus feature to allow to overcome limitations of message size.

```

Comment:
:+1: 
Agree on adding that underlying transport is the message size limiting factor. As for easily switching transports, that's not `DataBus` feature, but NServiceBus in general :)